source/production/F0.Minesweeper.Components/F0.Minesweeper.Components.csproj

```diff
@@ -4,6 +4,11 @@
 		<TargetFramework>net5.0</TargetFramework>
 	</PropertyGroup>
 
+	<ItemGroup>
+	  <None Remove="Game.razor.css" />
```

Comment:
This right here is also a point I want to mention. Already teasered it in our Slack channel as well with the incorrect setup of the components library project

Reply:
Hmmm, I guess the `Microsoft.NET.Sdk.Razor` project SDK does not support CSS isolation.
Perhaps only the `Microsoft.NET.Sdk.Web` project SDK.
Should we switch? - or perhaps don't use CSS isolation in our components lib and use the classic approach via `*.css` files in the `wwwroot` folder?

Reply:
I wouldn't really like it if we go away from scoped css. Because doing it the classic way is the same as style isolation without the benefits that the defined classes can only be used inside your component. 
And since it seems to just be an issue with Visual Studio itself that those lines are added I rather would ignore this for now and remove the unnecessary lines in the csproj file manually.